src/base-input.js

```diff
@@ -84,6 +86,8 @@ export default class BaseInput extends LitElement {
       label:                   { type: String },
       name:                    { type: String },
       type:                    { type: String },
+      format:                  { type: String },
```

Comment:
Had a side chat with @geoffrich regarding the forming [API standard](https://github.com/AlaskaAirlines/WC-Generator/issues/105) and we can't come up with a good reason to NOT extend the native HTML spec. 

Instead of creating a new attribute for this, why not extend the `type` attribute? Instead of describing a new attribute in the API, we simply add another option to the already existing `text`, `password`, etc...  options.

To that end, I think we have the opportunity to redefine the UI for `type="date"` when we get the green light on the UI for this. Given the designs we are seeing, we will never be using the native `type="date"`.

Reply:
I like that idea. I patched in a change - check it out and let me know if that matches up to what you are thinking.